.github/ISSUE_TEMPLATE/new_analyzer.md

```diff
@@ -14,6 +14,8 @@ assignees: ''
 ## Type of analyzer
 **this can be observable, file, and docker**
 
+**Please Note:** If the analyzer is free, Please add it in the `FREE_TO_USE_ANALYZERS` playbook in `playbook_config.json`
```

Comment:
ah one last thing, this should be put here: https://github.com/intelowlproject/IntelOwl/blob/master/.github/release_template.md. Can you move that please?

Reply:
yessir let me do that quickly